lib/vcapi.js

```diff
@@ -210,6 +198,14 @@ export async function processExchange({req, res, workflow, exchangeRecord}) {
           expectedChallenge
         });
 
+        // validate enveloped VP after verification
+        if(presentationSchema && isEnvelopedVP) {
+          validateVerifiablePresentation({
+            schema: presentationSchema.jsonSchema,
+            presentation: verifyResult?.presentationResult ?? {}
```

Comment:
Is `presentationResult` presentation? Or a JSON object that may / may not have a `presentation` in it?

We need to add a positive and negative test to confirm that this works and doesn't regress in the future.